recipes/perl-text-roman/meta.yaml

```
@@ -0,0 +1,25 @@
+package:
+  name: perl-text-roman
+  version: "3.5"
+
+source:
+  url: https://cpan.metacpan.org/authors/id/S/SY/SYP/Text-Roman-3.5.tar.gz
+  md5: 1f6b09c0cc1f4425b565ff787a39fd83
+
+build:
+  noarch: generic
+  number: 2
+
+requirements:
+  build:
+    - make
+  host:
+    - perl
+    - perl-module-build
+  run:
+    - perl
+
+about:
+  home: https://metacpan.org/pod/Text::Roman
+  license: perl_5
```

```suggestion
  license: Artistic-1.0-Perl
```
https://spdx.org/licenses/Artistic-1.0-Perl.html